ctapipe/calib/camera/calibrator.py

```diff
@@ -53,19 +53,17 @@ class CameraCalibrator(TelescopeComponent):
         The name of the ImageExtractor subclass to be used for image extraction
     """
 
-    data_volume_reducer_type = create_class_enum_trait(
+    data_volume_reducer_type = ComponentName(
         DataVolumeReducer, default_value="NullDataVolumeReducer"
     ).tag(config=True)
 
     image_extractor_type = TelescopeParameter(
```

Comment:
This doesn't list the possible values. Also happens for other List traits, e.g. ShowerProcessor. Maybe implement a subclass of List ListComponentName which also returns the subclasses of the trait in the help string and use this for the ShowerProcessor and as the superclass of TelescopeParameter

Reply:
Yes, I suggest calling it `ComponentNameList` though (it's a list of component names).  The current help string looks like this:
```
ShowerProcessor(Component) options
----------------------------------
--ShowerProcessor.reconstructor_types=<list-item-1>...
    The stereo geometry reconstructors to be used.
    Default: ['HillasReconstructor']
```
Which is not so useful as you don't see what options are available. This is the same issue as #1874 for TelescopeParameters.